src/CL/Slack/Model/Attachment.php

```diff
@@ -257,4 +262,20 @@ public function getFields()
     {
         return $this->fields;
     }
+    
+    /**
+     * @param array Valid values for mrkdwn_in are: ["pretext", "text", "fields"]. Setting "fields" will enable markup formatting for the value of each field
+     */
+    public function setMrkdwnIn(Array $mrkdwnIn)
```

Comment:
I could, but one of the features right in the readme is 
> Payloads and responses follow the same definitions as described in the official documentation (with a few exceptions where I think it would make a better distinction).

which I liked. Granted, you have a disclaimer, and where methods are concerned i'd tend to agree, but I don't think actual property names should be changed, otherwise you'd end up trying to compare a specific property and find them not matching up since they getter/setter is different. What do you think?